### PR TITLE
Executor timout behaviour

### PR DESCRIPTION
**Pull request by BolaNasr**
ExecutorSSH and ExecutorLocal are following the same timeout behaviour #9

### CODE REVIEW

**Comment by zaibon** [comment]
What's the reason behind this PR?

**Comment by BolaNasr** [comment]
fix this issue 
[Executor behaviour](https://github.com/threefoldtech/jumpscale_core/issues/9)

File: Jumpscale/tools/executor/ExecutorSSH.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def id(self):
     def executeRaw(self, cmd, die=True, showout=False):
         return self.sshclient.execute(cmd, die=die, showout=showout)
 
-    def execute(self, cmds, die=True, checkok=False, showout=True, timeout=0, env={}, asScript=True, sudo=False, shell=False):
+    def execute(self, cmds, die=True, checkok=False, showout=True, timeout=1000, env={}, asScript=True, sudo=False, shell=False):
```
Comment (zaibon) [inline Jumpscale/tools/executor/ExecutorSSH.py]: I wouldn't change the default value of timeout cause it change the behavior of the method.

Comment (BolaNasr) [reply Jumpscale/tools/executor/ExecutorSSH.py]: ok just i added it in sshclient.execute

Comment (zaibon) [reply Jumpscale/tools/executor/ExecutorSSH.py]: forgot to push ?

Comment (BolaNasr) [reply Jumpscale/tools/executor/ExecutorSSH.py]: Done

**Pull request merged**